tests/qchem/of_tests/test_decompose.py

```diff
@@ -6,6 +6,9 @@
 
 from pennylane import qchem
 
+openfermion = pytest.importorskip("openfermion")
+openfermionpyscf = pytest.importorskip("openfermionpyscf")
```

Comment:
@soranjh note that this will skip the entire file if openfermion is not installed. If this is not what you want, I recommend instead moving this inside the test functions.

Reply:
(and same for other test files that weren't previously skipping if OF wasn't available)

Reply:
I know that this approach is a bit aggressive. However, the important tests all depend on openfermion and pyscf. Also, there are some tests that depend on other functions that need these libs. There are also functions that depend on these libs for not good reason. Overall, the whole non-diff workflow tests need to be modified and updated which I prefer to do in a dedicated PR.

Reply:
Okay, as long as there is a ticket or issue for the additional work that needs to be done!

Another option could be to modify the tests so that all OpenFermion tests are in a single file, but not sure if this makes sense or not.

Reply:
Yes, that makes sense. The plan is to collect the OF-dependent tests in one place.

Reply:
Or add a TODO into the test file

Reply:
TODO added. There are also stories on Shortcut to address this.